### DISCUSSION TITLE
CPU Temperature - None

### DISCUSSION THREAD

**Discussion by Terence-D** [timmo001/system-bridge#2890] [Help] [answered]
Hey there, I've been trying to figure out what I'm doing wrong, but haven't had any luck.  Please let me know if this is the wrong area to ask / missing any information

Issue - CPU Temperature shows up as "None" even though sensors reports the temperature correctly.
System - Ubuntu 22.04
Version of System Bridge: 3.11.2
CPU: Intel Core i7-2600K 3.4 GHz Quad-Core Processor
Motherboard: Asus P8Z68-V PRO ATX LGA1155 Motherboard

python3 -m systembridgecli data cpu | grep temp
| None               | 1.70568e+09 | temperature                        |

sensors outputs:
nct6776-isa-0290
Adapter: ISA adapter
Vcore:           1.14 V  (min =  +0.00 V, max =  +1.74 V)
in1:             1.01 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
AVCC:            3.30 V  (min =  +2.98 V, max =  +3.63 V)
+3.3V:           3.30 V  (min =  +2.98 V, max =  +3.63 V)
in4:           1000.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in5:             2.04 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in6:             1.02 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
3VSB:            3.41 V  (min =  +2.98 V, max =  +3.63 V)
Vbat:            3.34 V  (min =  +2.70 V, max =  +3.63 V)
fan1:             0 RPM  (min =    0 RPM)
fan2:          2115 RPM  (min =    0 RPM)
fan3:             0 RPM  (min =    0 RPM)
fan4:             0 RPM  (min =    0 RPM)
fan5:             0 RPM  (min =    0 RPM)
SYSTIN:         +25.0°C  (high =  +0.0°C, hyst =  +0.0°C)  ALARM  sensor = thermistor
CPUTIN:         -60.0°C  (high = +80.0°C, hyst = +75.0°C)  sensor = thermal diode
AUXTIN:         +24.5°C  (high = +80.0°C, hyst = +75.0°C)  sensor = thermistor
PECI Agent 0:   +18.0°C  (high = +80.0°C, hyst = +75.0°C)
                         (crit = +88.0°C)
PCH_CHIP_TEMP:   +0.0°C
PCH_CPU_TEMP:    +0.0°C
PCH_MCH_TEMP:    +0.0°C
intrusion0:    ALARM
intrusion1:    ALARM
beep_enable:   disabled

acpitz-acpi-0
Adapter: ACPI interface
temp1:        +27.8°C  (crit = +99.0°C)
temp2:        +29.8°C  (crit = +99.0°C)

coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +30.0°C  (high = +80.0°C, crit = +98.0°C)
Core 0:        +30.0°C  (high = +80.0°C, crit = +98.0°C)
Core 1:        +29.0°C  (high = +80.0°C, crit = +98.0°C)
Core 2:        +27.0°C  (high = +80.0°C, crit = +98.0°C)
Core 3:        +26.0°C  (high = +80.0°C, crit = +98.0°C)

**Comment by Ekomak** [comment]
I have the same issue with same Ubuntu version 22.04 and sensors command works for me too.
Can't find a workaround to this issue.

**Comment by timmo001** [comment]
Can you view data and add the output of sensors. You should hopefully have a temperature value in there, its just the case of mapping it for the setup you have
![image](https://github.com/timmo001/system-bridge/assets/28114703/2cb837e5-e9b5-4cda-aa44-0799f3a0cfc6)

**Reply by Ekomak** [reply]
Thanks for your Help, 
When I view data, I get this output. Temperatures data are of course visible. Hope I could have help with this and you could give me information  about 

> its just the case of mapping it for the setup you have

`{
  "fans": {
    "atk0110": [
      [
        "CPU FAN Speed",
        1203
      ],
      [
        "CHASSIS FAN Speed",
        1112
      ],
      [
        "CHASSIS FAN 2 Speed",
        633
      ],
      [
        "POWER FAN Speed",
        934
      ]
    ]
  },
  "temperatures": {
    "atk0110": [
      [
        "CPU Temperature",
        44,
        60,
        95
      ],
      [
        "MB Temperature",
        32,
        45,
        75
      ]
    ],
    "k10temp": [
      [
        "",
        0,
        70,
        99.5
      ]
    ],
    "nouveau": [
      [
        "",
        33,
        95,
        105
      ]
    ]
  },
  "windows_sensors": null
}`

**Reply by timmo001** [reply, marked as answer]
Added in https://github.com/timmo001/system-bridge-backend/pull/117

**Reply by Ekomak** [reply]
Thanks @timmo001. As I install the .deb version of system bridge, I'll wait for the next .deb release to update front and backend. I don't want to mess up with my current installation. I'll send feed back after that. Thanks again.

**Reply by Ekomak** [reply]
Latest version 4.1.10  works like a charm. Thank you !!!
